pkg/resource/logging-config/alloy/logging.alloy.template

```diff
@@ -69,17 +69,20 @@ discovery.relabel "kubernetes_pods" {
 	}
 
 	rule {
-		source_labels = ["__meta_kubernetes_namespace"]
+		source_labels = ["instance"]
```

Comment:
I think we need to handle that differently as this part contains both discovery.relabel that predates pod logs and loki.relabel the future right?

Reply:
Yeah right, we need to support both versions

Reply:
Added a condition and use the new approach only for podlogs

Reply:
Nice. Do you think it would not make sense to add the condition to the whole `discovery/loki.relabel` sections as discovery targets prepodlogs and the other targets pod logs? Otherwise i'm good